### ISSUE TITLE
Audio sample rate of live stream mis-displayed

### ISSUE THREAD

**Issue by Vangelis66** [WendyH/hdsdump#5]
Hi Wendy

OS: Windows Vista SP2 32bit, fully patched + post-EOL updates (for WS2008)
.NET Framework: 4.6.1 (32bit) + all Security and Quality updates for it. 
hdsdump: v2.0.1.7 (latest)

I am using the following batch file:  
```cmd
hdsdump -m "http://as-hds-ww-live.akamaized.net/pool_7/live/bbc_radio_two/bbc_radio_two.isml/bbc_radio_two-audio=128000.f4m" -p | "F:\Applications\VLC Portable\VLC 2.1.x Portable (zip)\vlc-2.1.5\vlc.exe" --file-caching=30000 -

pause
exit
```
to listen to the AdobeHDS live stream of [BBC Radio 2](http://www.bbc.co.uk/radio/player/bbc_radio_two).
Here is a screenshot of things working: 

![wrongsr](https://user-images.githubusercontent.com/9669492/29245893-a12a5156-7ff1-11e7-9395-e861a3387cc0.jpg)

First discrepancy is the sampling rate (SR) reported by the CLI: `Audio: AAC 44 kHz Stereo`
VLC, on the other hand, correctly reports the SR to be **48 kHz**. I am 100% sure the SR=48kHz, the BBC have switched to that years ago, they said so at the time, only their MP3 Icecast live streams and their MP3 podcasts have remained at 44.1kHz (for compatibility reasons with old hardware...). 
Plus, if the live stream is downloaded as an .flv file (instead of being piped to VLC), then again MediaInfo correctly reports a SR of 48kHz. 

If hdsdump can't calculate the SR correctly, it must be patched accordingly, or, if not possible, do away altogether with displaying an (incorrect) SR value...

Second thing is (_which should certainly deserve a separate issue_, but.... ), when I exit VLC after a while, 
the command prompt window remains there, unresponsive; it has to be closed via the red "X" button; it looks as though "pause" and "exit" inside my batch file aren't being honoured... 
When using `php AdobeHDS.php` instead of `hdsdump` inside the batch file, exiting VLC leaves a functional CPW ("pause" has been executed): 

![adobephp](https://user-images.githubusercontent.com/9669492/29246134-9b7eb0b0-7ff8-11e7-81c5-b3a63729e404.jpg)

which, of course, closes by pressing any key... 

 Hopefully, you'll be able to address both issues... 
Many thanks for all the efforts spent so far for this magnificent tool :+1:

**Comment by WendyH** [comment]
Vangelis66, tnx 4 reporting. 
Fixed. Check the [2.0.1.8 version](https://github.com/WendyH/hdsdump/releases/tag/2.0.1.8).

P.S.: 48000Hz is out of flv specification... but its happens. Now showing more info from metadata of video.

**Comment by Vangelis66** [comment]
@WendyH wrote: 

> Fixed. Check the 2.0.1.8 version.

This is splendid! I can verify that **both issues** reported have been fixed 👍. 
However, while the 2.0.1.8 release did fix those things for me, **_it broke_** (:-1:) some other: 
The following batch file is used to dump 30s worth of the [BBC One](http://www.bbc.co.uk/iplayer/live/bbcone) live stream: 
```cmd
hdsdump -m "http://vs-hds-uk-live.akamaized.net/pool_30/live/bbc_one_hd/bbc_one_hd.isml/bbc_one_hd-pa4=128000-video=1604032.f4m" -t 30 -o BBCOneHD_Live_1732k-540p25_test.flv

pause
```
The AdobeHDS stream is fully (manifest+fragments) **geo-blocked** from overseas, you'd have to either use a whitelisted UK VPN or a reliable UK HTTP proxy (in the latter case, use: 
```cmd
hdsdump -px proxyhost:proxyport -fp -m "http://vs-hds-uk-live.akamaized.net/pool_30/live/bbc_one_hd/bbc_one_hd.isml/bbc_one_hd-pa4=128000-video=1604032.f4m" -t 30 -o BBCOneHD_Live_1732k-540p25_test.flv

pause
```
). Well, with previous version 2.0.1.7 of hdsdump, recording the TV stream works fine (albeit a wrong audio SR of 44kHz is reported): 

![2017-working](https://user-images.githubusercontent.com/9669492/29254102-9239566a-8096-11e7-9f20-68367d720804.jpg)

But with new version 2.0.1.8, **recording fails**:

![2018-broken](https://user-images.githubusercontent.com/9669492/29254114-de426880-8096-11e7-8d6d-6fbcb38c8b07.jpg)

(Both tests using the same UK VPN!)
I was unable to create a meaningful debug log, but here it is: 
[hdsdump.log](https://github.com/WendyH/hdsdump/files/1221033/hdsdump.log.txt)
... Must be something inside https://github.com/WendyH/hdsdump/commit/43259db that broke it, am suspecting the "**more audio & video info**" part of it...

Please help, if you can... BTW, they have now made signing-in compulsory in BBC iPlayer (for UK users), 
having hdsdump work is one way to evade that...

Thanks in advance!

**Comment by WendyH** [comment]
Sorry. That's my bad. Released [2.0.1.9](https://github.com/WendyH/hdsdump/releases/tag/2.0.1.9).

**Comment by Vangelis66** [comment]
Congrats **WendyH** ! **BBC One Live** works again as expected! Much obliged, really 🥇 ... 
I prefer using _hdsdump_ to _AdobeHDS.php_, because in the latter case one has to frequently update php, 
which receives monthly security and performance updates... 

I think it's time I closed this issue... 
Cheers
